manticore/platforms/evm.py

```diff
@@ -919,7 +927,7 @@ def assemble(asmcode, offset=0):
                 ...
                 "\x60\x60\x60\x40\x52\x60\x02\x61\x01\x00"
         '''
-        return ''.join(map(lambda x: x.bytes, EVMAsm.assemble_all(asmcode, offset=offset)))
+        return ''.join([x.bytes for x in EVMAsm.assemble_all(asmcode, offset=offset)])
```

Comment:
brackets `[...]` are redundant (as in https://github.com/trailofbits/manticore/pull/784/files?diff=unified#r175897327)